.github/workflows/documentation.yml

```diff
@@ -182,6 +182,15 @@ jobs:
         with:
           cmd: install --frozen-lockfile
 
+      - name: Cache Docusaurus Build
+        uses: actions/cache@v4
```

Comment:
kindly use action's hash

Reply:
I forgot to remove this section
This ws a test and cache usage is not worth